App/Navigation/PhotosNavigation.js

```diff
@@ -2,21 +2,23 @@ import React from 'react'
 import { StackNavigator, TabNavigator, TabBarBottom } from 'react-navigation'
 import { Image } from 'react-native'
 import TextilePhotos from '../Containers/TextilePhotos'
-import InfoView from '../Containers/InfoView'
 
 import styles, {headerTintColor} from './Styles/NavigationStyles'
+import AccountNavigation from './AccountNavigation'
```

Comment:
Unused I think.

Reply:
It is, it's part of the root nav at the bottom